otamodel/OTA_HotelProductNotifRQ.go

```diff
@@ -71,57 +131,64 @@ type StayDateRange struct {
 	End      string `json:"End,omitempty"`
 }
 
-//HotelProductRatePlan **
-type HotelProductRatePlan struct {
-	RatePlanCode  string          `json:"RatePlanCode,omitempty,required"`
-	RatePlanName  string          `json:"RatePlanName,omitempty,required"`
-	RatePlanType  string          `json:"RatePlanType,omitempty"`
-	CurrencyCode  string          `json:"CurrencyCode,omitempty,required"`
-	BaseOccupancy string          `json:"BaseOccupancy,omitempty,required"`
-	Descriptions  Descriptions    `json:"Descriptions,omitempty,required"`
-	StayDateRange []StayDateRange `json:"StayDateRange,omitempty,required"`
-}
-
-//HotelProductRatePlans **
-type HotelProductRatePlans struct {
-	RatePlan *[]HotelProductRatePlan `json:"RatePlan,omitempty,required"`
-}
-
 //PrepaymentPolicy **
 type PrepaymentPolicy struct {
 	EffectiveFrom string `json:"EffectiveFrom,omitempty"`
 }
 
 //TPAExtensions **
 type TPAExtensions struct {
-	PrepaymentPolicy PrepaymentPolicy `json:"PrepaymentPolicy,omitempty"`
+	PrepaymentPolicy *PrepaymentPolicy `json:"PrepaymentPolicy,omitempty"`
 }
 
 //GuaranteePayment **
 type GuaranteePayment struct {
```

Comment:
I think same object is already declared in common.go with type PaymentPoliciesGuaranteePayment

Reply:
Modified